docs/static/ds6/skin.css

```diff
@@ -809,6 +809,110 @@ span.inline-notice {
     outline-color: white;
 }
 */
+.pagination {
+  font-size: 0;
```

Comment:
Why font size 0?

Reply:
This is because `<a>` tag adds a little space with with a new line (browser interpretation). This resets the browser styles

https://stackoverflow.com/questions/6902237/css-unwanted-spacing-between-anchor-tag-elements/30254541

Reply:
Maybe we should see if we can do this another way? From the comments on that answer:

>Be careful with this code because it might be interpreted by web crawlers as abuse since you are explicitly hiding text inside an element containing links. Its true that the links itself are visible thanks to a higher specificity rule but still web crawlers see a bunch of links inside a container which tells all of its ancestors to hide their textual content.

Reply:
So should I change it back?